artemis/src/main/java/tech/pegasys/artemis/cli/options/LoggingOptions.java

```diff
@@ -54,9 +55,10 @@
   @CommandLine.Option(
       names = {LOG_DESTINATION_OPTION_NAME},
       paramLabel = "<LOG_DESTINATION>",
-      description = "Whether a logger is added for the console, the log file, or both",
+      description =
+          "Whether a logger is added for the console, the log file, or both (Valid values: ${COMPLETION-CANDIDATES})",
       arity = "1")
-  private String logDestination = DEFAULT_LOG_DESTINATION;
+  private LoggingDestination logDestination = LoggingDestination.get(DEFAULT_LOG_DESTINATION);
```

Comment:
Wouldn't it make more sense to just use:
```suggestion
  private LoggingDestination logDestination = LoggingDestination.DEFAULT_BOTH;
```

Reply:
just means its not set off the default - i can if you prefer...

Reply:
The `DEFAULT_LOG_DESTINATION` constant could also be of type `LoggingDestination`.  Although I really don't know why we've extracted all this stuff as constants and will likely do a pass through to inline them all when I get a chance.  It's so much easier to read in besu where the strings are used directly in the `@Option` annotations.